### ISSUE TITLE
`inspec json` command does not populate the attribues/inputs for the profile

### ISSUE THREAD

**Issue by aaronlippold** [mitre/inspec#9]
< fill in with references >

**Comment by Amndeep7** [comment]
Need to do research to see if this is still the case.

If it is, @aaronlippold is this still something we should fix?

**Comment by aaronlippold** [comment]
Yes if it's still the case. This would relate to your schema standardization clean up
